From c2f8e185938863082efbd525fabef22d6b719cb3 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov@fedoraproject.org>
Date: Sun, 15 Aug 2021 10:27:37 +0100
Subject: [PATCH 145/249] zdtm: sort import lines

These changes have been auto-generated with:

    pip3 install isort autoflake
    isort -rc -sl zdtm.py
    autoflake --remove-all-unused-imports -i zdtm.py
    isort -rc -m 3 zdtm.py

Signed-off-by: Radostin Stoyanov <rstoyanov@fedoraproject.org>
---
 test/zdtm.py | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

diff --git a/test/zdtm.py b/test/zdtm.py
index 9ad9ab24e..bb4a4c3b9 100755
--- a/test/zdtm.py
+++ b/test/zdtm.py
@@ -1,5 +1,10 @@
 #!/usr/bin/env python
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals
+)
 
 import argparse
 import atexit
@@ -14,6 +19,7 @@ import random
 import re
 import shutil
 import signal
+import socket
 import stat
 import string
 import struct
@@ -21,13 +27,12 @@ import subprocess
 import sys
 import tempfile
 import time
-import socket
-from builtins import (input, int, open, range, str, zip)
-
-import pycriu as crpc
+from builtins import input, int, open, range, str, zip
 
 import yaml
 
+import pycriu as crpc
+
 # File to store content of streamed images
 STREAMED_IMG_FILE_NAME = "img.criu"
 
@@ -1933,7 +1938,7 @@ class Launcher:
 
         if opts['report'] and (opts['keep_going'] or self.__total == 1):
             global TestSuite, TestCase
-            from junit_xml import TestSuite, TestCase
+            from junit_xml import TestCase, TestSuite
             now = datetime.datetime.now()
             att = 0
             reportname = os.path.join(report_dir, "criu-testreport.tap")
-- 
2.35.1

